rules/windows/process_creation/win_susp_crackmapexec_execution.yml

```diff
@@ -19,31 +19,18 @@ logsource:
     product: windows
 detection:
     selection:
-        - CommandLine|contains|all:
+        CommandLine|endswith:
             # cme/protocols/smb/wmiexec.py (generalized execute_remote and execute_fileless)
-            - 'cmd.exe /Q /c'
-            - '1> \\\\'
-            - '\'
-            - '\\'
-            - '2>&1'
-        - CommandLine|contains|all:
+            - 'cmd.exe /Q /c * 1> \\\\*\\*\\* 2>&1'
```

Comment:
in this specific example (as well as in the other similar ones) we've tried to get rid of the asterisks in the detection logic, as they could be misleading for the people that work with the raw sigma rules and for some (partially/or unsupported) backends. 
I agree that it looks bad in the normal YAML list, but we can use the following notation to solve the issue:

```
        - CommandLine|contains|all: [ 'cmd.exe /Q /c', '1> \\\\', '\', '\\', '2>&1' ]
```

Would it solve the issue? Are you OK with this approach? If yes, we can update the rules using it.

Reply:
Generally I'm totally fine with the notation as list across multiple lines. This was the only rule where I've reverted the change for a special reason. The rule is derived from source code, which means that the command line appears exactly as provided in the original rule. Making the pattern parts positionally independent as with the change makes it prone to false positives.

Reply:
I see. Thank you for your clarification!